### ISSUE TITLE
Failed to launch on 21.10

### ISSUE THREAD

**Issue by sesser** [keshavbhatt/PixelFx#2]
Installed from snap.
```
(pixelfx:30979): dbind-WARNING **: 16:28:14.176: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-ngm5hd7GGP: No such file or directory
Gtk-Message: 16:28:14.194: Failed to load module "canberra-gtk-module"
Gtk-Message: 16:28:14.194: Failed to load module "canberra-gtk-module"
QApplication: invalid style override passed, ignoring it.
    Available styles: Windows, Fusion
/usr/share/libdrm/amdgpu.ids: No such file or directory
Fontconfig warning: FcPattern object weight does not accept value [80 215)
[1]    30979 segmentation fault (core dumped)  pixelfx
```
I have the canberra-gtk modules installed.

```
[130] % apt list --installed |grep libcanberra

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

libcanberra-gtk-module/impish,now 0.30-7ubuntu2 amd64 [installed]
libcanberra-gtk0/impish,now 0.30-7ubuntu2 amd64 [installed,automatic]
libcanberra-gtk3-0/impish,now 0.30-7ubuntu2 amd64 [installed,automatic]
libcanberra-gtk3-module/impish,now 0.30-7ubuntu2 amd64 [installed,automatic]
libcanberra-pulse/impish,now 0.30-7ubuntu2 amd64 [installed]
libcanberra0/impish,now 0.30-7ubuntu2 amd64 [installed,automatic]
```

**Comment by keshavbhatt** [comment]
no the app doesn't require anything to be installed. its a self contained package i.e, it pack everything that is required to run. 

this issue is old and solution is here 
https://github.com/keshavbhatt/red/issues/1#issuecomment-846354883

**Comment by keshavbhatt** [comment]
`rm -rf ~/snap/pixelfx` should do the trick the latest version should not have this problem.
